usr.sbin/makefs/ffs.c

```diff
@@ -882,6 +882,7 @@ ffs_write_file(union dinode *din, uint32_t ino, void *buf, fsinfo_t *fsopts)
 	struct inode	in;
 	struct buf *	bp;
 	ffs_opt_t	*ffs_opts = fsopts->fs_specific;
+	struct vnode vp = { fsopts, NULL };
```

Comment:
This is an interesting change made in NetBSD. Instead, can we simply pass `fsopts` as an argument all the way down to `bread`? I understand the abstraction of using a vnode pointer, but it seems unnecessary.